spec/index.html

```diff
@@ -1053,10 +1053,38 @@ <h2>Internet Media Type, File Extension and Macintosh File Type</h2>
     <dt>Required parameters:</dt>
     <dd>None</dd>
     <dt>Optional parameters:</dt>
-    <dd><code>version</code> — this parameter is optional but SHOULD be present when using RDF 1.2-specific features.
+    <dd><dl>
+      <dt><code>version</code></dt><dd>This parameter is optional but SHOULD be present when using RDF 1.2-specific features.
```

Comment:
I think "included" signals intentional addition or use better than "present".

```suggestion
      <dt><code>version</code></dt><dd>This parameter is optional but SHOULD be included when using RDF 1.2-specific features.
```

Reply:
I don't feel strongly about it but I slightly prefer "present". Any other opinion?

Reply:
I'm inclined toward "included".